src/legacy/ui/public/kfetch/kfetch.ts

```diff
@@ -50,7 +51,7 @@ export const addInterceptor = (interceptor: Interceptor) => interceptors.push(in
 
```

Comment:
Since the CSV download is immediate, and is not JSON, I needed to opt-out of the res.json() behavior of kfetch.